src/main/java/nl/hsac/fitnesse/util/iban/BEIban.java

```diff
@@ -0,0 +1,232 @@
+package nl.hsac.fitnesse.util.iban;
+
+import nl.hsac.fitnesse.util.RandomUtil;
+
+/**
+ * Generates a Belgium IBAN.
+ */
+
```

Comment:
In IntelliJ, '**/' and enter above a method will give you the javadoc comment. Pretty useful.